scripts/save_org_product_data_daily_off_pro.pl

```diff
@@ -107,6 +107,13 @@ ($org_id)
 	)->next;
 
 	my $number_of_products = $org_data->{number_of_products} // 0;
+
+	# Using off-query to count products with a specific owners_tags seems very slow
```

Comment:
@john-gom @alexgarel for some reason the queries to count products with a specific owners_tags are very slow on off-query :
```

Processing organization 9/5021: nestle-waters
nestle-waters	75	75	time: 1.26603510975838
Processing organization 10/5021: nestle
nestle	0	0	time: 1.26369819045067
Processing organization 11/5021: org-carrefour
org-carrefour	0	0	time: 1.25258129835129
Processing organization 12/5021: unilever-bangladesh
unilever-bangladesh	0	0	time: 1.2691158875823
Processing organization 13/5021: unilever-france-gms
unilever-france-gms	2369	1159	time: 1.27603560686111
Processing organization 14/5021: unilever-france-rhd
unilever-france-rhd	149	145	time: 1.27778738737106

```